### DISCUSSION TITLE
Kaboom and Vue

### DISCUSSION THREAD

**Discussion by TortoiseLeaf** [replit/kaboom#539] [Q&A]
Hey everyone, I'm trying to run a kaboom instance inside a Vue web-app and while it's working, it's a little buggy (or I'm not doing something properly!)

Biggest issues are, regardless of using cdn or installing with npm:

- eslint will throw an error about the Kaboom syntax (removing and reinstalling eslint fixes the issue weirdly)
- once the game is booted up on the server on page A, the Kaboom instance shows on every other page too
- (less severe but still a problem) I can't quite manage to position the instance on a page, for example to center it

Would really appreciate any thoughts or guidance,
Much appreciated!

**Comment by lajbel** [comment]
Hey, sorry for the delay, it's a bit strange... Make sure you have the latest versions
